pennylane/ops/cv.py

```diff
@@ -566,6 +603,10 @@ def _heisenberg_rep(p):
         M[1 : 2 * N + 1, 1 : 2 * N + 1] = S
         return M
 
+    def adjoint(self, do_queue=False):
+        U = self.parameters[0]
+        return Interferometer(U.conj().T, wires=self.wires, do_queue=do_queue)
```

Comment:
Note: this will break using `qml.adjoint()` with `strawberryfields.tf` - we will need to make this framework agnostic

Reply:
I mentioned this already in an earlier comment. Wonder why it wasn't changed before merging in :thinking:

Reply:
Maybe the comment was missed?

Reply:
Nope, it was definitely seen (and lead to this change). I'm just wondering if there is some technical reason why we can't use `qml.math` inside the operator class?

Reply:
> I'm just wondering if there is some technical reason why we can't use qml.math inside the operator class?

Not that I am aware. I have a long term plan to use it within the operator class (so that we can do away with the separate `tf_ops.py`, `autograd_ops.py`, etc.

Reply:
The reason why is because `qml.math.conj` didn't exist before. I made a PR to add it and fixed the affected ops.
https://github.com/PennyLaneAI/pennylane/pull/1143